orangecontrib/text/datasets/KKS_1.001.tab

```diff
@@ -0,0 +1,13 @@
+Source	Created	Category	ID	Content	Source URL	User

```

Comment:
I know the file name is a reference to the article but it has no meaning to the end-user. Could we think of a better one? Also, could we add this dataset with more than just 10 documents?